### PR TITLE
chore: update svelte-vite-ts to svelte 4 and vite 5

### PR DESCRIPTION
**Pull request by AtofStryker**
Updates svelte-vite-ts to vite 5 and svelte 4. Needs to go into release 14 branch due to how documentation is phrased on docs.cypress.io

### CODE REVIEW

File: svelte-vite-ts/package.json [hunk 1]
```diff
@@ -8,21 +8,19 @@
     "build": "vite build",
     "preview": "vite preview",
     "check": "svelte-check --tsconfig ./tsconfig.json",
-    "cypress:run": "cypress run --config video=false --component"
+    "cypress:run": "cypress run --config --component"
```
Comment (mschile) [inline svelte-vite-ts/package.json:11]:
```suggestion
    "cypress:run": "cypress run --component"
```

**Comment by mschile** [comment]
@AtofStryker, I received the following error running the tests:

```
Failed to load PostCSS config
Your configFile threw an error from: cypress.config.js

We stopped running your tests because your config file crashed.
```

**Comment by AtofStryker** [comment]
> @AtofStryker, I received the following error running the tests:
> 
> ```
> Failed to load PostCSS config
> Your configFile threw an error from: cypress.config.js
> 
> We stopped running your tests because your config file crashed.
> ```

weird I can't reproduce it. I tried doing a fresh node_modules install as well and nothing. any ideas how to recreate?

**Review by mschile — APPROVED** [review]

**Comment by AtofStryker** [comment]
> > @AtofStryker, I received the following error running the tests:
> > ```
> > Failed to load PostCSS config
> > Your configFile threw an error from: cypress.config.js
> > 
> > We stopped running your tests because your config file crashed.
> > ```
> 
> weird I can't reproduce it. I tried doing a fresh node_modules install as well and nothing. any ideas how to recreate?

we were able to reproduce this in node 18. doing this worked https://github.com/QwikDev/qwik/issues/836#issuecomment-1763157670

**Pull request merged**